pwndbg/color/memory.py

```diff
@@ -24,6 +25,28 @@
 )
 
 
+# Ex: convert and colorize address 0x7ffff7fcecd0 to string `0x7ffff7fcecd0 (_dl_fini)`
+# If no symbol, it is omitted
+def get_address_and_symbol(address: int) -> str:
+    symbol = pwndbg.gdblib.symbol.get(address) or None
+    if symbol:
+        symbol = f"{address:#x} ({symbol})"
+    return get(address, symbol)
+
+
+# Ex: convert and colorize address to symbol if it can be resolved, else return hex string of address
+def get_address_or_symbol(address: int) -> str:
+    return attempt_colorized_symbol(address) or get(address)
+
+
+# Convert address to colorized symbol (if symbol is there), else None
```

Comment:
Can you use Python docstrings for function documentation instead of comments?

Reply:
On it - I'll go through all the changes and convert all function/method/member variables to use docstring's in the cases I didn't.